src/SearchViewController.swift

```diff
@@ -207,9 +208,7 @@ Ensure you are using a unique, valid shortcut.
 
     func closeApp() {
         clearFields()
-        if promptValue == "" {
```

Comment:
This change isn't necessary, just saw this could be simplified as `PipeListProvider.doAction` terminates the app before this code is reached -- `promptValue == ""` will always evaluate to true.